tensorflow/core/kernels/depthtospace_op.cc

```diff
@@ -17,13 +17,12 @@ limitations under the License.
 
 #define EIGEN_USE_THREADS
 
+#include "tensorflow/core/kernels/depthtospace_op.h"
```

Comment:
The order of includes changed according to clang-formatting tool with `-style=google`